src/components/Layout/Header/Header.tsx

```diff
@@ -8,19 +9,25 @@ const Header: React.FC = () => {
   return (
     <AppBar position='static' id='header' className={classes.header}>
       <Toolbar className={classes.toolbar}>
-        <Logo imgWidth={55} />
+        <Link to='/' className={classes['logo-link']}>
+          <Logo imgWidth={55} />
+        </Link>
         <Box className={classes.actions}>
-          <IconButton size='large' color='inherit'>
-            <Badge badgeContent={4} color='error'>
-              <Favorite />
-            </Badge>
-          </IconButton>
+          <Link to='/wish-list' className={classes['nav-link']}>
```

Comment:
Create a separate enum for your routes and use them as constants in your components

```suggestion
          <Link to={Routes.WISH_LIST} className={classes['nav-link']}>
```

Reply:
fixed